pkg/kudoctl/kudoinit/manager/manager.go

```diff
@@ -105,12 +105,12 @@ func generateDeployment(opts kudoinit.Options) *appsv1.StatefulSet {
 		},
 		ObjectMeta: metav1.ObjectMeta{
 			Namespace: opts.Namespace,
-			Name:      "kudo-controller-manager",
+			Name:      opts.ManagerName,
```

Comment:
I'm not sure I understand why this was done for this PR... but it feels strange to have it in `Options`.   It should be a const not a part of struct

Reply:
as a constant we would like use it for all the "kudo-controller-manager" references in code... looks like a number of tests

Reply:
after reading the full PR :)... I see that there is a constant... but we are using it in many places.... can we?

Reply:
and can we remove it from options?

Reply:
I put into the options because we wire them anyway. But you're right, I don't see us making these hard-coded strings changeable in the near future so there is no reason to put them into the `Options`. Removed 👍